app/assets/stylesheets/components/_modal-dialogue.scss

```diff
@@ -0,0 +1,146 @@
+$govuk-modal-margin: govuk-spacing(6);
+$govuk-modal-close-button-size: 40px;
+$govuk-modal-z-index: 1000;
+
+// Wrapper that the modal scrolls within
+.app-c-modal-dialogue {
+  display: block;
+  position: fixed;
+  z-index: $govuk-modal-z-index;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  outline: 0;
+}
+
+.app-c-modal-dialogue__box {
+  display: block;
+  position: fixed;
+  z-index: $govuk-modal-z-index - 10;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  overflow-y: auto;
+  border: 0;
+
+  @include govuk-media-query($from: tablet) {
+    position: relative;
+    top: inherit;
+    right: inherit;
+    bottom: inherit;
+    left: inherit;
+    width: auto;
+    max-width: $govuk-page-width * 2 / 3;
+    height: auto;
+    margin: $govuk-modal-margin auto;
+    border: $govuk-border-width-form-element solid $govuk-input-border-colour;
+  }
+
+  @include govuk-font($size: 19);
+  @include govuk-text-colour;
+  @include govuk-focusable;
+}
+
+.app-c-modal-dialogue__box--wide {
+  @include govuk-media-query($from: tablet) {
+    max-width: $govuk-page-width;
+  }
+}
+
+.app-c-modal-dialogue__overlay {
+  position: fixed;
+  z-index: $govuk-modal-z-index - 20;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  opacity: .8;
+  background: govuk-colour("black");
+}
+
+.app-o-template__body--modal {
+  overflow: hidden;
+}
+
+.app-o-template__body--blur {
+  .govuk-skip-link,
+  .govuk-header,
+  .govuk-phase-banner,
+  .govuk-back-link,
+  .govuk-footer,
+  .govuk-main-wrapper {
+    filter: blur(2px);
+  }
+}
+
+.app-c-modal-dialogue__header {
+  padding: 6px govuk-spacing(3) 8px;
+  color: govuk-colour("white");
+  background: govuk-colour("black");
+}
+
+.app-c-modal-dialogue__logotype-crown {
+  margin-right: 1px;
```

Comment:
Why does this need a 1px margin right?

Reply:
I carried this from the header component where the crown was followed by the gov.uk text. Removed it from here as it doesn't help.